staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -33,14 +34,17 @@ const (
 	// LevelRequest provides Metadata level of auditing, and additionally
 	// logs the request object (does not apply for non-resource requests).
 	LevelRequest Level = "Request"
-	// LevelResponse provides Request level of auditing, and additionally
+	// LevelRequestResponse provides Request level of auditing, and additionally
 	// logs the response object (does not apply for non-resource requests).
-	LevelResponse Level = "Response"
+	LevelRequestResponse Level = "RequestResponse"
```

Comment:
They are ordered anyway. To be consistent this must be LevelMetaRequestResponse. No I don't like that.